### ISSUE TITLE
`Shift` keyboard button isn't working

### ISSUE THREAD

**Issue by svipas** [software-mansion/radon-ide#835]
**Description**

If I type into input fields using `Shift` it isn't working, instead of `U` it returns `u`, same goes for symbols like `!`, etc. I have to use keyboard from the simulator.

**Environment**

- Version of currently used Radon IDE: 1.0.0
- Version of React Native/Expo used in the project: 51 (Expo) and 0.74 (RN)

**Comment by svipas** [comment]
Seems like after reloading VS Code it started to work again, not sure what the issue was.

**Comment by ovidiu-deliu** [comment]
In my case neither VSCode restart or `fn + shift` don't solve the issue. It's a bit clunky as neither `CMD + V` doesn't work for pasting data from an outer context respective to the device, so I have to paste manually (iOS hold click and wait for the pop-up options)

**Comment by maciekstosio** [comment]
Hi, thanks for the feedback. We're investing in the problem, but it is a bit tricky as it's related to communication with the iOS simulator, which is not documented. You can paste something from the clipboard using long press and paste, but it's not always reliable.

**Comment by ovidiu-deliu** [comment]
In my case for an iPhone 15 Pro simulator - the shift key simply started recognized all of a sudden. The `CMD + V` command still doesn't work but at least I can use shift for text-inputs to input uppercased letters and symbols.

If needed I can provide any requested data/config for debugging

**Comment by maciekstosio** [comment]
@ovidiu-deliu Yeah, it stops working randomly, probably, it depends on changing input preferences during the keyboard transition on the simulator. 

> If needed I can provide any requested data/config for debugging

Thanks, events are sent to the simserver properly, so for now, It's not necessary. I can reproduce the problem reliably, but I haven't found a proper solution yet. We're working on it.

> The CMD + V command still doesn't work

This won't work. When you're copying `CMD + C`, we copy the selected content to the device clipboard. You should be able to long-press the input and select `paste` on the device. It works, although when I tested it, it wasn't always reliable.

**Comment by ovidiu-deliu** [comment]
To add more feedback to the issue - It seems that Caps Lock could fix the issue, but it is behaving interestingly.

In my case (as you mentioned @maciekstosio) the keyboard randomly transitions between two states:
 - it works normally (shift works as expected)
 - shift doesn't work + Caps Lock behaves differently

The CapsLock behavior goes as following:
-> typing lower case 
-> press Caps Lock -> weird state where most of the letters do not input and it seems like there's a command state, because pressing H minimizes the app (tried other letters, they do not seem to do anything - only inputs letters "RTYOPSDJKLNM", the others are ignored)
-> press again Caps Lock -> typing upper case
-> press again Caps Lock -> weird state

Also - I can somehow transition between the "with-shift" and "without-shift" states also seemingly randomly

**Comment by maciekstosio** [comment]
@ovidiu-deliu Caps Lock behaves weirdly because, in contrary to other buttons, clicking Caps-Lock (down and up) sends just one event. So despite it beeing pressed and released, keyboard treats it as it is hold for the entrie time. That supposed to be fiexed in https://github.com/software-mansion/radon-ide/pull/986, but unfortunetly it's separate problem.

**Comment by jarredwitt** [comment]
Currently dealing with the shift key not working in the iOS sim. 

IDE Version: 1.3.1
iPhone 15 Pro sim

The shift key works just fine in Android although the numpad does not (may be a separate issue). On iOS the numpad works great, but the shift key is not working. It's making it incredibly frustrating to test out auth flows that require emails as I can type shift+2 to get the @ symbol on iOS.

**Comment by ovidiu-deliu** [comment]
Can confirm that the issue was indeed fixed in the latest release (currently on 1.6.0) - thanks team!
